src/Cache/FileCacheStorage.php

```diff
@@ -55,7 +55,7 @@ public function load(string $key, string $variableKey)
 			}
 
 			return $cacheItem->getData();
-		})($key, $variableKey);
```

Comment:
instead of defining a function and immediately invoking it, I wen't with closure-params which feels easier to read for me